sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/ServiceBusClientBuilderJavaDocCodeSamples.java

```diff
@@ -44,6 +33,32 @@ public void instantiateReceiverAsync() {
         receiver.receiveMessages().blockFirst(Duration.ofSeconds(1));
     }
 
+    public void instantiateSessionReceiver() {
+        // BEGIN: com.azure.messaging.servicebus.session.receiver.async.client.instantiation
+        // Retrieve 'connectionString', 'topicName' and 'subscriptionName' from your configuration.
+        ServiceBusClientBuilder builder = new ServiceBusClientBuilder()
+            .connectionString(connectionString);
+        // Create a session receiver.
+        ServiceBusSessionReceiverAsyncClient sessionReceiver = builder
```

Comment:
The builder methods should be chained to demonstrate a better pattern when not sharing connections.